idb/postgres/postgres.go

```diff
@@ -1095,7 +1103,9 @@ func (db *IndexerDb) yieldAccountsThread(req *getAccountsRequest) {
 			account.PendingRewards = 0
 		} else {
 			// TODO: pending rewards calculation doesn't belong in database layer (this is just the most covenient place which has all the data)
-			proto, ok := config.Consensus[req.blockheader.CurrentProtocol]
+			// todo: replace config.Consensus. config.Consensus map[protocol.ConsensusVersion]ConsensusParams
+			// temporarily cast req.blockheader.CurrentProtocol(string) to protocol.ConsensusVersion
+			proto, ok := config.Consensus[protocol.ConsensusVersion(req.blockheader.CurrentProtocol)]
```

Comment:
what's another way to get consensus params?

Reply:
We'll need a new consensus package, which copies all these settings over. I saw you already created an issue for that :+1: